_layouts/default.html

```diff
@@ -86,8 +94,8 @@
 
       <footer class="text-center text-muted">
         <hr/>
-        Last updated {{ site.time | date: "%B %d, %Y" }}. <br/>
-	The source code is on <a href="https://github.com/callison-burch/data-structures-and-algorithms">github <i class="fa fa-github-alt"></i></a>.
+        <script type="text/javascript">document.write("Last updated " + document.lastModified + ".");</script> <br/>
```

Comment:
I think it makes more sense to let Jekyll staticly generate the last update time. We could rename this to be something like "Last generated"

@lyze Thoughts?